airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -83,9 +92,8 @@ def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Ite
             days_of_data_storage=self.days_of_data_storage,
         )
 
-    @staticmethod
-    def get_date_params(stream_slice: Mapping[str, Any], cursor_field: str, end_date: pendulum.datetime = None, time_unit: str = "months"):
-        end_date = end_date or pendulum.yesterday()
+    def get_date_params(self, stream_slice: Mapping[str, Any], cursor_field: str, end_date: pendulum.datetime = None, time_unit: str = "months"):
+        end_date = end_date or pendulum.today(tz=self.time_zone)
```

Comment:
shouldn't this be yesterday based on the comment above?

Reply:
Yep, I returned it to yesterday.